src/main/resources/application.yml

```diff
@@ -73,7 +73,16 @@ workspace:
     uri: ${env.db.host}/${env.db.ws.name}
     username: ${env.db.ws.user}
 
-terra.common:
-  tracing:
-    stackdriverExportEnabled: ${env.tracing.exportEnabled}
-    samplingRate: ${env.tracing.samplingRate}
+terra:
+  common:
+    tracing:
+        stackdriverExportEnabled: ${env.tracing.exportEnabled}
+        samplingRate: ${env.tracing.samplingRate}
+    stairway:
+      cluster-name-suffix: workspace-stairway
+      force-clean-start: false
+      max-parallel-flights: 500
```

Comment:
Probably more than we need. I'd suggest 50. It is per node and we probably don't want more than 150 threads competing for access to the backend database server.